components/edit-collective/sections/Host.js

```diff
@@ -363,12 +360,12 @@ class Host extends React.Component {
               {selectedOption === 'selfHost' && LoggedInUser && (
                 <Flex justifyContent="space-between" alignItems="flex-end" mt={3}>
                   <Box>
-                    <Button bsStyle="primary" type="submit" onClick={() => this.changeHost({ id: collective.id })}>
```

Comment:
It's good to keep the `type="submit"` if already set, makes things explicit in the DOM